app/src/main/res/menu/menu_main.xml

```diff
@@ -3,23 +3,30 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
+    <item
+        android:id="@+id/menu_search"
+        android:title="@string/menu_search"
+        android:icon="@drawable/ic_baseline_search_24"
+        app:showAsAction="ifRoom"
+        app:actionViewClass="android.widget.SearchView"/>
```

Comment:
Do you know whether it is possible to hide the activity name when the search expands? It looks a bit odd as it kind of cuts the name. So far I have not seen such a behavior in other apps (at least I did not notice).